### PR TITLE
add recognition for clare

### PR DESCRIPTION
**Pull request by evykassirer**
addresses #13

### CODE REVIEW

**Comment by arshiamufti** [comment]
I like it without the "...heh".

Also, just a suggestion--you might want to ask clare if she'd rather her website be linked instead of twitter.

**Comment by evykassirer** [comment]
will do!

File: _pages/about.md [hunk 1]
```diff
@@ -27,4 +27,12 @@ Anna, Arshia, and Evy attended [HelloCon](http://hellocon.net) in Toronto, which
 
 The star in StarCon was inspired by the star (asterisk) that is the [wildcard character](http://whatis.techtarget.com/definition/wildcard-character) used in many programming tools. This character represents one or more other characters of any type, and our hope is that StarCon can represent many different types of people and topics in technology.
 
+---
+
+## Our Logo
+
+Our logo was designed by the amazing and talented [Clare Ng](https://twitter.com/clare_n9)! It's a *light*bulb with a star in the middle because we're a *light*ning talk conference.
```
Comment (annalorimer) [inline _pages/about.md]: Did we ask Clare if we can link to this?

Comment (annalorimer) [inline _pages/about.md]: Also, maybe explain *why* it's a star in the middle.

**Comment by arshiamufti** [comment]
lgtm 👍

**Comment by evykassirer** [comment]
how's this?

**Comment by arshiamufti** [comment]
I like it. 🚢  it!

**Pull request merged**